ape_solidity/compiler.py

```diff
@@ -163,8 +172,8 @@ def get_import_remapping(self, base_path: Optional[Path] = None) -> Dict[str, st
                 if not cached_manifest_file.exists():
                     raise CompilerError(f"Unable to find dependency '{suffix}'.")
 
-                manifest_dict = json.loads(cached_manifest_file.read_text())
-                manifest = PackageManifest(**manifest_dict)
+                # NOTE: Purposely skip Pydantic validation here.
+                manifest = PackageManifest(**json.loads(cached_manifest_file.read_text()))
```

Comment:
I think this still does validation, the suggestion from the docs is to use `.construct`
https://pydantic-docs.helpmanual.io/usage/models/#creating-models-without-validation

Reply:
weird, it doesn't seem like it does! But good to know

Reply:
Done!

Reply:
Welp, so this causes the `Source` objects to never get serialized as classes and that causes mypy issues, so I reverted this.

There seems to be a difference between using the normal `init` versus `parse_obj()` where the init still gives me classes but does not run validators